### DISCUSSION TITLE
Insert content through backend/lambda function

### DISCUSSION THREAD

**Discussion by Jkense** [liveblocks/liveblocks#1358] [Help]
Hi all,

I just came across liveblocks and it looks great. I have been working on trying to implement collaborative features on my TipTap editor. My problem is that TipTap offers very limited documentation, unlike liveblocks. Before I decide to use liveblocks I would like to have one question answered though:

Would it be possible to insert/implement text through an API using liveblocks? I want to use a GCF to insert content into my YDoc.

Hope someone can help me out,
Thanks,
Jasper

**Comment by Malki** [comment]
I have the same question about liveblocks. Did you manage to know if this is possible?
